packages/turf-kinks/types.ts

```diff
@@ -1,5 +1,8 @@
+/*
```

Comment:
For this package, what import was causing problems?

Reply:
```import { kinks } from "./index.js";``` bombs with a complaint about "Property 'default' does not exist on type ..." of whatever index.ts is trying to import. I can get IDE typing + the build to work, or the tests, but (seemingly) not both.

Can try to see if I can get the test re-enabled. Or if you're happy with the changes to the PR so far, we can merge and add a couple of issues to revisit this, possibly with PRs to the upstream projects.

Reply:
@mfedderly managed to get the type tests enabled again. Turns out the `tsc ... types.ts` command doesn't refer to the tsconfig.json in the current directory. Setting --module and --moduleResolution to node16 on the command line (to match tsconfig.shared.json) fixed that problem. Can't find a way to avoid the import wrappers entirely, however building, testing and type tests are working.